[quant][graphmode] Move processing code to prepare_script

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#38669 [quant][graphmode] Move processing code to prepare_script**

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21623385](https://our.internmc.facebook.com/intern/diff/D21623385)

jerryzh168 committed:
[quant][graphmode] Move processing code to prepare_script

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar commented on torch/quantization/_quantize_script.py:
```diff
@@ -22,9 +22,13 @@ def script_qconfig_dict(qconfig_dict):
+    if any(not isinstance(x, str) for x in qconfig_dict.keys()):
         raise ValueError('qconfig_dict should contain names(str) as keys.')
     scripted_qconfig_dict = script_qconfig_dict(qconfig_dict)
+    if not is_dynamic:
```
Do we need this condition here? Previously we used to call this for dynamic as well.

jerryzh168 replied:
do we need conv bn folding for dynamic quantization?

jerryzh168 replied:
cc @raghuramank100

jerryzh168 committed:
Update on "[quant][graphmode] Move processing code to prepare_script"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21623385](https://our.internmc.facebook.com/intern/diff/D21623385)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Move processing code to prepare_script"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21623385](https://our.internmc.facebook.com/intern/diff/D21623385)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Move processing code to prepare_script"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21623385](https://our.internmc.facebook.com/intern/diff/D21623385)

[ghstack-poisoned]
